src/drc/api/utils.py

```diff
@@ -12,3 +16,22 @@ def get_absolute_url(url_name: str, uuid: str) -> str:
     domain = Site.objects.get_current().domain
     protocol = 'https' if settings.IS_HTTPS else 'http'
     return f'{protocol}://{domain}{path}'
+
+
+def merge_files(part_files, file_dir, file_name) -> str:
+    if not os.path.exists(file_dir):
+        os.makedirs(file_dir)
```

Comment:
there's an option `ignore=True` or something like that, that you can use to make the dirs if it doesn't exist yet. That allows you to remove the `os.path.exists` check.